apps/docs/src/docs/components/demo/PopoverContent.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <!-- #region template -->
+  <div class="d-flex gap-2">
+    <BButton id="popover-props" variant="primary">Using properties</BButton>
+    <BPopover
+      target="popover-props"
+      title="Prop Examples"
+      content="Embedding content using properties is easy"
+    />
+    <BButton id="popover-slots" variant="primary">Using slots</BButton>
+    <BPopover target="popover-slots">
+      <template #title>Content via Slots</template>
+      Embedding content <span class="text-danger">using slots</span> affords you
+      <em>greater <strong>control.</strong></em> and basic HTML support.
```

Comment:
```suggestion
      <em>greater <strong>control.</strong></em> and HTML support.
```
All the html is usable, since we don't need to jump through hoops to support it (like it was in bv2 and directives)